apps/sms/index.html

```diff
@@ -378,6 +378,18 @@ <h1 data-l10n-id="message-report"></h1>
       -->
     </div>
 
+    <div id="contact-report-tmpl" class="hide">
+      <!--
+      <li>
```

Comment:
In this case, I think we don't want it. In the other cases, we have a `role=option` attribute in the `<a>` and that's why we use `role=presentation` on the `<li>`; from the screenreader everything looks like a `<select><option>` element.

But here we don't have the same behavior; here we show a real list. That's why I think we should keep the default meeting for `<li>`: a list item.

@eeejay, can you please confirm?